test/e2e/cmd/handlers-test

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+SPOT_TEST_PATH="http://$HOSTNAME:$AEMM_PORT/latest/meta-data/spot"
+SPOT_TEST_PATH_TRAILING="http://$HOSTNAME:$AEMM_PORT/latest/meta-data/spot/"
+SPOT_IA_TEST_PATH_TRAILING="http://$HOSTNAME:$AEMM_PORT/latest/meta-data/spot/instance-action/"
+SPOT_TT_TEST_PATH_TRAILING="http://$HOSTNAME:$AEMM_PORT/latest/meta-data/spot/termination-time/"
+
+function test_subpaths() {
```

Comment:
may be a good test to add is with a req path - '/latest/meta-data/network/interfaces/macs/<mac>/' to test `ipv4-associations/` with trailing slash